target/scripts/startup/setup-stack.sh

```diff
@@ -379,7 +379,7 @@ function _setup_ldap
     [[ ${FILE} =~ ldap-aliases ]] && export LDAP_QUERY_FILTER="${LDAP_QUERY_FILTER_ALIAS}"
     [[ ${FILE} =~ ldap-domains ]] && export LDAP_QUERY_FILTER="${LDAP_QUERY_FILTER_DOMAIN}"
     [[ ${FILE} =~ ldap-senders ]] && export LDAP_QUERY_FILTER="${LDAP_QUERY_FILTER_SENDERS}"
-    _replace_by_env_in_file 'LDAP_' "${FILE}"
+    [[ -f ${FILE} ]] && _replace_by_env_in_file 'LDAP_' "${FILE}"
```

Comment:
If you add your guard here, then the failure log/panic would be not be raised? Makes more sense to communicate the failure?

```suggestion
    _replace_by_env_in_file 'LDAP_' "${FILE}"
```

Reply:
Done in [20c16eb](https://github.com/docker-mailserver/docker-mailserver/pull/3045/commits/20c16eb0526548a4e343b678eb97f6dde44e1216)